cross-chain/sei/hardhat.config.ts

```diff
@@ -28,15 +28,30 @@ const config: HardhatUserConfig = {
         settings: {
           optimizer: {
             enabled: true,
-            runs: 1000,
+            runs: 200, // Reduced from 1000 to prioritize bytecode size over gas efficiency
           },
         },
       },
     ],
+    overrides: {
+      // Optimize L1BTCDepositorNttWithExecutor for minimal bytecode size
+      "contracts/cross-chain/wormhole/L1BTCDepositorNttWithExecutor.sol": {
+        version: "0.8.17",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1, // Minimal runs to minimize bytecode size
+          },
```

Comment:
NIT: I know it's not strictly part of this PR, but does it make sense to have the shared settings in a single object to avoid repetition?